vnext/ReactUWP/Views/Image/ReactImage.h

```diff
@@ -48,7 +48,7 @@ namespace react {
 
       // Public Properties
       ImageSource Source() { return m_imageSource; }
-      winrt::fire_and_forget Source(ImageSource source);
+      winrt::fire_and_forget Source(ImageSource source, std::string bundleRootPath);
```

Comment:
Add bundleRootPath to ImageSource instead.

Reply:
Done.